packages/tailwindcss/src/utilities.ts

```diff
@@ -431,6 +432,14 @@ export function createUtilities(theme: Theme) {
             value = desc.handleBareValue(candidate.value)
             if (!value?.includes('/') && candidate.modifier) return
           }
+
+          if (value === null && !negative && desc.staticValues) {
+            let fallback = desc.staticValues[candidate.value.value]
+            if (fallback) {
+              if (candidate.modifier) return
+              return fallback
+            }
+          }
```

Comment:
```suggestion
          if (value === null && !negative && desc.staticValues && !candidate.modifier) {
            let fallback = desc.staticValues[candidate.value.value]
            if (fallback) {
              return fallback
            }
          }
```

Reply:
Diff isn't super clear, but essentially move the `if (candidate.modifier) return` into the main `if`